benchmarks/Neo.Benchmarks/config.json

```diff
@@ -0,0 +1,71 @@
+{
+  "ApplicationConfiguration": {
```

Comment:
Is this config still rquired now that you're using the `ProtocolSettings.Default`?

Reply:
yes